inc/CalculatorStrings.h

```diff
@@ -16,16 +16,66 @@ namespace calculator
     struct CalculatorStrings
     {
         /**
-         * @brief prompt passed to user before retrieving input from them
+         * @brief ""
+         * 
+         */
+        static const std::string EMPTY_STRING;
+
+        /**
+         * @brief " = "
+         * 
+         */
+        static const std::string EQUAL;
+
+        /**
+         * @brief Beginning of error message
          * 
          */
-        static const std::string MAIN_PROMPT; 
```

Comment:
Everywhere Ditto A struct is a class in C++ (mostly) and static const std::string in this way is subject to static order initialization fiasco see https://meowni.ca/posts/static-initializers/  bascially you may get away with it but it is bad practice.  Hint: static constexpr char MY_CONST_VAR[] = "some string"; which I think I have seen in production code.